frontend/src/pages/modelRegistry/screens/ModelVersions/ModelVersions.tsx

```diff
@@ -29,7 +29,9 @@ const ModelVersions: React.FC<ModelVersionsProps> = ({ tab, ...pageProps }) => {
   const loadError = mvLoadError || rmLoadError;
   const loaded = mvLoaded && rmLoaded;
   const navigate = useNavigate();
-  const inferenceServices = useMakeFetchObject(useInferenceServices(undefined, rmId));
+  const inferenceServices = useMakeFetchObject(
+    useInferenceServices(undefined, rmId, preferredModelRegistry?.metadata.name),
```

Comment:
@ppadti you're passing the MR name as the 3rd argument here instead of the 4th argument, which I think would make it be interpreted as the version ID. You may need an `undefined` before it so it's in the 4th position.